api/src/cli/index.ts

```diff
@@ -30,6 +32,19 @@ export async function createCli(): Promise<Command> {
 	program.command('start').description('Start the Directus API').action(startServer);
 	program.command('init').description('Create a new Directus Project').action(init);
 
+	// Security
+	const securityCommand = program.command('security');
+	securityCommand
+		.command('key:generate')
+		.description('Generate the app key')
+		.option('--show', 'Show the key without updating the .env file')
```

Comment:
I think the only thing we can do is make this the default, and remove the writing to .env altogether